apps/admin/app/api/partners/top-connector/route.ts

```diff
@@ -0,0 +1,14 @@
+import { getLastWeek } from '@packages/dates/utils';
+
+import { respondWithTSV } from 'lib/nextjs/respondWithTSV';
+import { getScoutsForTopConnectorPartner } from 'lib/partners/getScoutsForTopConnectorPartner';
+
+export const dynamic = 'force-dynamic';
+
+export async function GET() {
+  const daysAgo = 14;
+  const lastWeek = getLastWeek();
```

Comment:
just fyi i dont think u need to put the last week in the export name, it seems like it would just be confusing

Reply:
Removed!